ingestion_tools/dataset_configs/10001.yaml

```diff
@@ -372,7 +372,7 @@ datasets:
       corresponding_author_status: true
       name: Judith B. Zaugg
     cell_strain:
-      id: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=284812
+      id: NCBITaxon:284812
```

Comment:
We should co-ordinate with the FE engineers about this change, as they use this id for constructing the link. 
For example: https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=284812 has the relevant value we are looking for. But https://www.ncbi.nlm.nih.gov/Taxonomy/Browser/wwwtax.cgi?id=NCBITaxon:284812  doesn't.

Reply:
Sounds good!